service/gcsutils/libtar2vhd/tar2vhd.go

```diff
@@ -80,7 +80,7 @@ func VHD2Tar(in io.Reader, out io.Writer, options *Options) (int64, error) {
 		return 0, err
 	}
 
-	mntFolder, err := ioutil.TempDir("", "mnt")
```

Comment:
Think you misread the docs:
```func TempDir(dir, prefix string) (name string, err error)```
TempDir creates a new temporary directory **in the directory dir** with a **name beginning with prefix** and returns the path of the new directory.

See example here: https://play.golang.org/p/bMEYqcs6zN .  The second parameter is for the directory name, not for the where the directory is placed in.